lib/PromAlertProxy/PromAlert.pm

```diff
@@ -0,0 +1,82 @@
+package PromAlertProxy::PromAlert;
```

Comment:
Fair! It used to be so because the previous version had a `PromAlert` and a `VOAlert` and (in a short-lived development branch) a `LPAlert`. Lone `Alert` is better.